src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4765,8 +4765,22 @@ internal static SessionStateAliasEntry[] BuiltInAliases
 
         internal const string DefaultPromptFunctionText = @"
 ""PS $($executionContext.SessionState.Path.CurrentLocation)$('>' * ($nestedPromptLevel + 1)) "";
+# .ForwardHelpTargetName about_Prompts
+# .SYNOPSIS
+# The PowerShell command prompt indicates
+# that PowerShell is ready to run a command.
+# .DESCRIPTION
+# To customize the prompt, write a new Prompt function.
+# The function is not protected, so you can overwrite it.
+# .OUTPUTS
+# The Prompt function must return an object.
+# As a best practice, return a string
+# or an object that is formatted as a string.
+# The maximum recommended length is 80 characters.
 # .Link
-# https://go.microsoft.com/fwlink/?LinkID=225750
+# https://go.microsoft.com/fwlink/?LinkId=2136843
+# .Link
+# about_Prompts
```

Comment:
Don't these help nodes require it to be all uppercase? Or am I misremembering. All the help docs I can recall  show them only with uppercase names (i.e., `.LINK`)

Reply:
This is the current declaration and it works.